dbms/src/Flash/Planner/ExecutorIdGenerator.h

```diff
@@ -78,6 +77,6 @@ class ExecutorIdGenerator
 
     UInt32 current_id = 0;
 
-    std::unordered_set<std::string_view> ids;
```

Comment:
amazing ...

Reply:
The lifetime of the string referenced by std::string_view only lives in `ExecutorIdGenerator::generate`.